unique_toolkit/unique_toolkit/language_model/service.py

```diff
@@ -44,22 +51,32 @@ def complete(
         Returns:
             LanguageModelResponse: The LanguageModelResponse object.
         """
-        return self._trigger_complete(
-            messages=messages,
-            model_name=model_name,
-            temperature=temperature,
-            timeout=timeout,
-            tools=tools,
-        )
+        options = self._add_tools_to_options({}, tools)
+        messages = messages.model_dump(exclude_none=True)
+        try:
+            response = unique_sdk.ChatCompletion.create(
+                company_id=self.state.company_id,
+                # TODO change or extend types in unique_sdk
+                model=model_name.name,  # type: ignore
+                messages=cast(
+                    list[unique_sdk.Integrated.ChatCompletionRequestMessage],
+                    messages,
+                ),
+                timeout=timeout,
+                temperature=temperature,
+                options=options,  # type: ignore
```

Comment:
There is no explicit mention of `options` in the `unique_sdk`. @unique-fabian are you aware of what this does and if it is needed?

Reply:
I discussed this with Fabian and also wondered that the parameter is not present in the sdk but somehow it works. ^^

Reply:
K thankyou. Let's keep this in mind and add it to the `sdk` explicitly.